activiti-api-process-model/src/main/java/org/activiti/api/process/model/events/BPMNTimerExecutionFailureEvent.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2018 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.api.process.model.events;
+
+public interface BPMNTimerExecutionFailureEvent extends BPMNTimerEvent {
```

Comment:
Can we rename it to `BPMNTimerFailedEvent`?

Reply:
Changed here: https://github.com/Activiti/activiti-api/pull/128/commits/bfacc62271726b78eaa753fe45c286c0ea1a8a42